### PR TITLE
app-emulation/docker: bump to coreos-1.11.2

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by mischief** [comment]
fixes coreos/bugs#1414

**Comment by marineam** [comment]
symlink is the wrong verson

File: app-emulation/docker/docker-9999.ebuild [hunk 1]
```diff
@@ -260,16 +249,13 @@ src_compile() {
...
-	#./man/md2man-all.sh || die "unable to generate man pages"
 }
 
 src_install() {
 	VERSION="$(cat VERSION)"
 	newbin "bundles/$VERSION/dynbinary/docker-$VERSION" docker
-	exeinto /usr/libexec/docker
-	newexe "bundles/$VERSION/dynbinary/dockerinit-$VERSION" dockerinit
+	#exeinto /usr/libexec/docker
+	#newexe "bundles/$VERSION/dynbinary/dockerinit-$VERSION" dockerinit
```
Comment (marineam) [inline app-emulation/docker/docker-9999.ebuild]: I'm guessing dockerinit is replaced by runc magic? Prune the old lines, not comment them.

Comment (mischief) [inline app-emulation/docker/docker-9999.ebuild]: it was leftover from gentoo tomfoolery.

File: app-emulation/docker/docker-9999.ebuild [hunk 2]
```diff
@@ -223,25 +226,11 @@ src_compile() {
...
-		DOCKER_BUILDTAGS+=" selinux"
-	fi
-	if use apparmor; then
-		DOCKER_BUILDTAGS+=' apparmor'
-	fi
-	if use journald; then
-		DOCKER_BUILDTAGS+=' journald'
-	fi
-
-	if has_version '<sys-fs/lvm2-2.02.110' ; then
```
Comment (marineam) [inline app-emulation/docker/docker-9999.ebuild]: not really applicable to us but if this check is gone should the DEPEND line now require at least this version?

Comment (mischief) [inline app-emulation/docker/docker-9999.ebuild]: i mistakenly removed it, @glevand added it for arm64 so i think it should be kept.

**Comment by marineam** [comment]
LGTM but hold off on merging until this passes in jenkins

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by crawford** [comment]
LGTM

**Comment by marineam** [comment]
How's early-docker looking? Does it need the option to use a different containerd socket?

**Comment by marineam** [comment]
@mischief ok, please add that to this PR

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
coreosrocks

**Comment by crawford** [comment]
LGTM

**Pull request merged**